kong/plugins/jwt/handler.lua

```diff
@@ -175,6 +181,15 @@ end
 function JwtHandler:access(conf)
   JwtHandler.super.access(self)
 
+  -- add uri whitelist
+  local method = ngx.var.request_method
+  local uri = ngx.var.uri
+  for _, v in ipairs(conf.uri_whitelist) do
+    if (v.method == method) and (v.uri == uri) then
+      return
+    end
+  end
+
```

Comment:
this logic does not belong in the plugin, please remove it (see my other comment)

Reply:
aleady remove!